### DISCUSSION TITLE
Unable to yarn install last version in docker

### DISCUSSION THREAD

**Discussion by xiangweizheng** [FlowiseAI/Flowise#1370] [Q&A]
Follow the https://docs.flowiseai.com/deployment/gcp to setup docker, but stucked in yarn install step when building docker image.
Error:
=> [11/13] RUN yarn install                                             876.7s
 => => # info There appears to be trouble with your network connection. Retryin
 => => # g...                                                                  
 => => # info There appears to be trouble with your network connection. Retryin
 => => # g...                                                                  
 => => # info There appears to be trouble with your network connection. Retryin
 => => # g...

**Comment by HenryHengZJ** [comment]
looks like proxy issue?
